### PR TITLE
New Post: Server Properties

### PR DESCRIPTION
**Pull request by KuroiJigoku**

### CODE REVIEW

File: _posts/minecraft/bedrock/2024-07-15-server-properties.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Configuring Bedrock Minecraft Server: Key Attributes of server.properties"
+category: 
+    - Bedrock
+    - Getting-started
+tags: Server
+description: "Explore the key attributes of the server.properties file to effectively configure your Minecraft server with this comprehensive guide."
+permalink: /minecraft/bedrock/server/server-properties
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-15-server-properties.md:9]: Author? Add `author: "Kuroi Jigoku"` on the next line

File: _posts/minecraft/bedrock/2024-07-15-server-properties.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "Configuring Bedrock Minecraft Server: Key Attributes of server.properties"
+category: 
+    - Bedrock
+    - Getting-started
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-15-server-properties.md:6]: The post only shows up in the Getting-started category, you should only add one category

File: _posts/minecraft/java/2024-07-14-server-properties.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "Configuring Java Minecraft Server: Key Attributes of server.properties"
+category: 
+    - Java
+    - Getting-started
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-14-server-properties.md:6]: The post only shows up in the Getting-started category, you should only add one category

File: _posts/minecraft/bedrock/2024-07-15-server-properties.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+tags: Server
+description: "Explore the key attributes of the server.properties file to effectively configure your Minecraft server with this comprehensive guide."
+permalink: /minecraft/bedrock/server/server-properties
+toc: false
+---
+
+## Introduction
+
+The `server.properties` file for a Bedrock Edition Minecraft server is a configuration file used to set various options and parameters for the server. This file allows server administrators to customize the server's behavior, such as setting the game mode, maximum player limit, difficulty level, server name, and more. By editing the server.properties file, you can control key aspects of the gameplay experience for all players on your Bedrock server.
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-15-server-properties.md:15]: I feel like you just asked chatgpt to write this introduction
Try making something like here: https://paper-chan.moe/paper-optimization/

File: _posts/minecraft/java/2024-07-14-server-properties.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+tags: Server
+description: "Explore the key attributes of the server.properties file to effectively configure your Minecraft server with this comprehensive guide."
+permalink: /minecraft/java/server/server-properties
+author: "Kuroi Jigoku"
+toc: false
+---
+## Introduction
+
+Understanding the server.properties file is crucial for configuring and customizing your Minecraft server. This guide highlights the essential attributes within the file, enabling you to optimize your server settings for the best gameplay experience
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-14-server-properties.md:15]: I feel like you just asked chatgpt to write this introduction
Try making something like here: https://paper-chan.moe/paper-optimization/

**Review by iamdeka — CHANGES_REQUESTED** [review]

File: _posts/minecraft/bedrock/2024-07-15-server-properties.md [hunk 6]
```diff
@@ -1,56 +1,35 @@
 ---
 layout: post
-title:  "Configuring Bedrock Minecraft Server: Key Attributes of server.properties"
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-15-server-properties.md]: This title is weird

File: _posts/minecraft/java/2024-07-14-server-properties.md [hunk 7]
```diff
@@ -1,11 +1,9 @@
 ---
 layout: post
-title:  "Configuring Java Minecraft Server: Key Attributes of server.properties"
-category: 
-    - Java
-    - Getting-started
+title:  "Configuring Minecraft Server: Key Attributes of server.properties"
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-14-server-properties.md:3]: This title is weird

File: _posts/minecraft/bedrock/2024-07-15-server-properties.md [hunk 8]
```diff
@@ -1,56 +1,35 @@
...
+description: "Explore the key attributes of the server.properties file to effectively configure your Bedrock Minecraft server with this comprehensive guide."
 permalink: /minecraft/bedrock/server/server-properties
+author: "Kuroi Jigoku"
 toc: false
 ---
 
 ## Introduction
 
-The `server.properties` file for a Bedrock Edition Minecraft server is a configuration file used to set various options and parameters for the server. This file allows server administrators to customize the server's behavior, such as setting the game mode, maximum player limit, difficulty level, server name, and more. By editing the server.properties file, you can control key aspects of the gameplay experience for all players on your Bedrock server.
+Understanding the server.properties file is crucial for configuring and customizing your Minecraft server. This guide highlights the essential attributes within the file, enabling you to optimize your server settings for the best gameplay experience
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-15-server-properties.md:14]: Dot?

File: _posts/minecraft/java/2024-07-14-server-properties.md [hunk 9]
```diff
@@ -10,7 +10,7 @@ toc: false
 ---
 ## Introduction
 
-This guide introduces the most common attributes used in the `server.properties` file, which may help you understand the usage of these attributes.
+Understanding the server.properties file is crucial for configuring and customizing your Minecraft server. This guide highlights the essential attributes within the file, enabling you to optimize your server settings for the best gameplay experience
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-14-server-properties.md:13]: Dot?

File: _posts/minecraft/bedrock/2024-07-15-server-properties.md [hunk 10]
```diff
@@ -11,7 +11,7 @@ toc: false
 
 ## Introduction
 
-This guide introduces the most common attributes used in the `server.properties` file, which may help you understand the usage of these attributes.
+Understanding the server.properties file is crucial for configuring and customizing your Minecraft server. This guide highlights the essential attributes within the file, enabling you to optimize your server settings for the best gameplay experience
 
 ## server.properties
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-15-server-properties.md:16]: I dont like how you just slap the table with the description of each attribute. At least make a dedicated section for commonly used ones and describe why they are used.

File: _posts/minecraft/java/2024-07-14-server-properties.md [hunk 11]
```diff
@@ -10,7 +10,7 @@ toc: false
...
 
 ## server.properties
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-14-server-properties.md:15]: I dont like how you just slap the table with the description of each attribute. At least make a dedicated section for commonly used ones and describe why they are used.

**Review by iamdeka — CHANGES_REQUESTED** [review]

File: _posts/minecraft/bedrock/2024-07-16-server-properties.md [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+### gamemode
+
+Possible Values: survival, creative, adventure  
+Default Value: survival  
+
+Defines the mode of gameplay.
+
+- If set to `survival`, players must gather resources, build structures, and survive against hostile mobs.
+- If set to `creative`, players have unlimited resources, can fly, and are not attacked by hostile mobs.
+- If set to `adventure`, players can interact with objects like levers and buttons but cannot break blocks.
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-16-server-properties.md:39]: I didnt mean to literally explain the most well known and obvious stuff 😭

File: _posts/minecraft/bedrock/2024-07-16-server-properties.md [hunk 13]
```diff
@@ -0,0 +1,188 @@
...
+- If set to `operator`, players have access to all commands and administrative privileges.
+
+---
+
+### chat-restriction
+
+Possible Values: None, Dropped, Disabled  
+Default Value: None  
+
+Allows control over chat functionality, enhancing server moderation and player experience.
```
Comment (iamdeka) [inline _posts/minecraft/bedrock/2024-07-16-server-properties.md:171]: It doesn't enhance anything literally 1984

**Review by iamdeka — APPROVED** [review]

File: _posts/minecraft/bedrock/2024-07-16-server-properties.md (hunk 13 above)
Comment (KuroiJigoku) [reply _posts/minecraft/bedrock/2024-07-16-server-properties.md:171]: i was drunk in code

File: _posts/minecraft/bedrock/2024-07-16-server-properties.md (hunk 12 above)
Comment (KuroiJigoku) [reply _posts/minecraft/bedrock/2024-07-16-server-properties.md:39]: 😭

File: _posts/minecraft/java/2024-07-16-server-properties.md [hunk 14]
```diff
@@ -0,0 +1,339 @@
...
+The maximum allowed size of the world radius, in blocks. This only affects the chunks that are generated when the world is initially created, and not the world border.
+
+---
+
+### motd
+
+Possible Values: String
+Default Value: A Minecraft Server
+
+The message of the day (MOTD) is displayed in the server list and provides a brief description or welcome message for players.
```
Comment (iamdeka) [inline _posts/minecraft/java/2024-07-16-server-properties.md:187]: You meticulously crafted a guide for this!
Add the {.info} thing telling users to refer to your guide.

**Comment by iamdeka** [comment]
I gave up halfway editing it 😃

**Comment by LiljaFi** [comment]
awesome

**Comment by KuroiJigoku** [comment]
dang 😶‍🌫️

**Comment by KuroiJigoku** [comment]
Bruh doesn't "author" have illegal characters now. like isn't it supposed to be a string _. cuz i put a whitespace..

**Comment by iamdeka** [comment]
> Bruh doesn't "author" have illegal characters now. like isn't it supposed to be a string _. cuz i put a whitespace..

should be fine

**Review by iamdeka — APPROVED** [review]

**Review by DaddyChimken — APPROVED** [review]

**Comment by iamdeka** [comment]
@Mocab look who's back

**Comment by KuroiJigoku** [comment]
:O

**Comment by Mocab** [comment]
halfway done 😩

**Review by Mocab — APPROVED** [review]
Get this out of my face

**Pull request merged**